### PR TITLE
DNS parser updated for schema 1.3

### PR DESCRIPTION
**Pull request by oshezaf**

### CODE REVIEW

File: Parsers/ASimDns/ProductParsers/VimDnsGcp.yaml [hunk 1]
```diff
@@ -56,7 +56,7 @@ ParserQuery: |
     | where resource_type_s == "dns_query"
     //  Pre-parsing filtering:
     | where
-      (eventtype=='lookup') // -- for now we support only lookup events
+    (eventtype in ('lookup', 'Query')) // -- for now we support only lookup events
```
Comment (YaronFruchtmann) [inline Parsers/ASimDns/ProductParsers/VimDnsGcp.yaml:59]: Applies to all vim files: I noticed you did not change the default value.  Do you find it risky?

File: Parsers/ASimDns/ProductParsers/vimDnsMicrosoftOMS.yaml [hunk 2]
```diff
@@ -68,65 +95,69 @@ ParserQuery: |
...
-    , response_has_any_prefix:dynamic=dynamic([]) , eventtype:string='lookup'
-    , disabled:bool=false
+    starttime:datetime=datetime(null), 
+    endtime:datetime=datetime(null),
+    srcipaddr:string='*',
+    domain_has_any:dynamic=dynamic([]),
+    responsecodename:string='*', 
+    response_has_ipv4:string='*',
+    response_has_any_prefix:dynamic=dynamic([]),
+    eventtype:string='*',
```
Comment (YaronFruchtmann) [inline Parsers/ASimDns/ProductParsers/vimDnsMicrosoftOMS.yaml:105]: did you mean to change that?

**Review by YaronFruchtmann — CHANGES_REQUESTED** [review]
I think you accidently changed one the default parameters values

File: Parsers/ASimDns/ProductParsers/VimDnsGcp.yaml (hunk 1 above)
Comment (oshezaf) [reply Parsers/ASimDns/ProductParsers/VimDnsGcp.yaml:59]: Thanks!

I update for consistent handling:
- Default is always Query
- lookup is accepted as a parameter and handled as Query.

**Pull request merged**